docs/source/reference/environment.rst

```diff
@@ -31,6 +31,14 @@ Here are the environment variables CuPy uses.
 |                                    | ``--generate-line-info``).                         |
 |                                    | It is disabled by default.                         |
 +------------------------------------+----------------------------------------------------+
+| ``CUPY_GPU_MEMORY_LIMIT``          | The amount of memory that can be allocated for     |
+|                                    | each device.                                       |
+|                                    | The value can be specified in absolute bytes or    |
+|                                    | fraction (e.g., ``"90%"``) of the total memory of  |
```

Comment:
Perhaps it's a matter of preference, but I prefer the notation like `0.9`, because `'%'` is a notation for human.

Reply:
I agree with you, but I didn't come up with a way to clearly distinguish size and fraction (e.g. `CUPY_GPU_MEMORY_LIMIT=1`).